dbt/adapters/athena/impl.py

```diff
@@ -278,10 +277,10 @@ def upload_seed_to_s3(
             s3_client.upload_file(tmpfile, bucket, object_name)
             os.remove(tmpfile)
 
-        return s3_location
+        return str(s3_location)
```

Comment:
Here, s3_location should already return string type no ?

Reply:
unfortunately I get the following error:
```
dbt/adapters/athena/impl.py:280: error: Returning Any from function declared to return "str"  [no-any-return]
```